roles/openshift_logging_fluentd/templates/fluent.conf.j2

```diff
@@ -42,6 +42,7 @@
   @include configs.d/openshift/filter-pre-*.conf
   @include configs.d/openshift/filter-retag-journal.conf
   @include configs.d/openshift/filter-k8s-meta.conf
+  @include configs.d/openshift/filter-k8s-events.conf
```

Comment:
Does this need to be conditionally included for cases where we are not deploying eventrouter?

Reply:
yes
either do it here, or do it in fluentd/run.sh (assuming there is a new env var added to the logging-fluentd ds and logging-mux dc) - then run.sh could replace with a 0 length file if not using the event router

Reply:
Since we agreed that the filter will be part of [ViaQ data model plugin](https://github.com/ViaQ/fluent-plugin-viaq_data_model), I will remove this from `fluent.conf` as soon as I have a PR there